src/core/config/transformer/rename_types.rs

```diff
@@ -65,6 +83,51 @@ impl Transform for RenameTypes {
                         }
                     }
                 }
+
+                // replace in interface.
+                type_.implements = type_
+                    .implements
+                    .iter()
+                    .filter_map(|interface_type_name| {
+                        lookup
+                            .get(interface_type_name)
+                            .cloned()
+                            .or(Some(interface_type_name.clone()))
```

Comment:
This is eagerly evaluated, use `or_else`

Reply:
using `unwrap_or`.